charts/milvus/templates/mixcoord-deployment.yaml

```diff
@@ -11,9 +11,18 @@ metadata:
 {{ include "milvus.ud.annotations" . | indent 4 }}
 
 spec:
+  {{- if ge (int .Values.mixCoordinator.replicas) 0 }}
```

Comment:
Sry, but I don't get it here. why is this `if` necessary? @Rachit-Chaudhary11

Reply:
It is available for all the components except the coordinators. Therefore I have added it.
For instance:
- https://github.com/zilliztech/milvus-helm/blob/master/charts/milvus/templates/datanode-deployment.yaml#L15
- https://github.com/zilliztech/milvus-helm/blob/master/charts/milvus/templates/indexnode-deployment.yaml#L14
- and so on

Reply:
@haorenfsa It is available for all the components except the coordinators. Therefore I have added it. 
For instance:
- https://github.com/zilliztech/milvus-helm/blob/master/charts/milvus/templates/datanode-deployment.yaml#L15
- https://github.com/zilliztech/milvus-helm/blob/master/charts/milvus/templates/indexnode-deployment.yaml#L14
- and so on

This is just a precautionary check.

Reply:
@haorenfsa It is available for all the components except the coordinators. Therefore I have added it. 
For instance:
- https://github.com/zilliztech/milvus-helm/blob/master/charts/milvus/templates/datanode-deployment.yaml#L15
- https://github.com/zilliztech/milvus-helm/blob/master/charts/milvus/templates/indexnode-deployment.yaml#L14
- and so on

This is just a precautionary check.

Reply:
@haorenfsa It is available for all the components except the coordinators. Therefore I have added it.
For instance:

https://github.com/zilliztech/milvus-helm/blob/master/charts/milvus/templates/datanode-deployment.yaml#L15
https://github.com/zilliztech/milvus-helm/blob/master/charts/milvus/templates/indexnode-deployment.yaml#L14
and so on
This is just a precautionary check.

Reply:
@haorenfsa It is available for all the components except the coordinators. Therefore I have added it.
For instance:

https://github.com/zilliztech/milvus-helm/blob/master/charts/milvus/templates/datanode-deployment.yaml#L15
https://github.com/zilliztech/milvus-helm/blob/master/charts/milvus/templates/indexnode-deployment.yaml#L14
and so on
This is just a precautionary check.

Reply:
ah, yes, it's for supporting hpa, #16